2024-08-29 20:17:25 align.main INFO : Reading netlist: /home/adair/Documents/CAD/characterization_script_sandbox_rundir/current_mirror_ota/current_mirror_ota.sp subckt=CURRENT_MIRROR_OTA, flat=0
2024-08-29 20:17:25 align.compiler.compiler INFO : Starting topology identification...
2024-08-29 20:17:25 align.compiler.user_const INFO : Reading constraint file: [PosixPath('/home/adair/Documents/CAD/characterization_script_sandbox_rundir/current_mirror_ota/current_mirror_ota.const.json')]
2024-08-29 20:17:25 align.compiler.compiler INFO : Completed topology identification.
2024-08-29 20:17:25 align.cmdline ERROR : Fatal Error. Cannot proceed
Traceback (most recent call last):
  File "/home/adair/Documents/CAD/venv/lib/python3.10/site-packages/align/cmdline.py", line 197, in parse_args
    return schematic2layout(**vars(arguments))
  File "/home/adair/Documents/CAD/venv/lib/python3.10/site-packages/align/main.py", line 181, in schematic2layout
    primitives = generate_primitives(primitive_lib, pdk_dir, primitive_dir, netlist_dir, blackbox_dir, scale)
  File "/home/adair/Documents/CAD/venv/lib/python3.10/site-packages/align/primitive/main.py", line 145, in generate_primitives
    generate_primitive_param(primitive, primitives, pdk_dir)
  File "/home/adair/Documents/CAD/venv/lib/python3.10/site-packages/align/primitive/main.py", line 173, in generate_primitive_param
    rc = modules.gen_param(subckt, primitives, pdk_dir)
  File "/home/adair/Documents/CAD/align/ALIGN-public/ALIGN-pdk-sky130/SKY130_PDK/gen_param.py", line 151, in gen_param
    assert int(mvalues[device_name]["NFIN"]) == int(mvalues[key]["NFIN"]), f"W should be same for all devices in {name} {mvalues}"
NameError: name 'name' is not defined
